e2e-tests/local-testnet.sh

```diff
@@ -407,6 +406,42 @@ rm -r ~/.relayer
 # initialize gorelayer
 rly config init
 
+# add chain configs
+
+    echo "{
```

Comment:
nit: formatting compared to the `echo` after this one. Move to the left.

Reply:
:thumbs-up: Good catch